x-pack/platform/plugins/shared/ml/public/application/explorer/anomaly_explorer_common_state.ts

```diff
@@ -42,65 +48,144 @@ export class AnomalyExplorerCommonStateService extends StateService {
     };
   }
 
-  constructor(private anomalyExplorerUrlStateService: AnomalyExplorerUrlStateService) {
+  constructor(
+    private anomalyExplorerUrlStateService: AnomalyExplorerUrlStateService,
+    private globalUrlStateService: UrlStateService<GlobalState>,
+    private mlJobsService: MlJobService
+  ) {
     super();
     this._init();
   }
 
-  protected _initSubscriptions(): Subscription {
-    return this.anomalyExplorerUrlStateService
-      .getPageUrlState$()
-      .pipe(
-        map((urlState) => urlState?.mlExplorerFilter),
-        distinctUntilChanged(isEqual)
-      )
-      .subscribe((v) => {
-        const result = {
-          ...this._getDefaultFilterSettings(),
-          ...v,
-        };
-        this._filterSettings$.next(result);
-      });
-  }
+  public readonly selectedGroups$: Observable<GroupObj[]> = this._selectedGroups$.pipe(
+    distinctUntilChanged(isEqual),
+    shareReplay(1)
+  );
 
-  public setSelectedJobs(explorerJobs: ExplorerJob[] | undefined) {
-    this._selectedJobs$.next(explorerJobs);
-  }
+  public readonly invalidJobIds$: Observable<string[]> = this._invalidJobIds$.pipe(
+    distinctUntilChanged(isEqual),
+    shareReplay(1)
+  );
 
-  public getSelectedJobs$(): Observable<ExplorerJob[]> {
-    return this._selectedJobs$.pipe(
-      filter((v) => Array.isArray(v) && v.length > 0),
-      distinctUntilChanged(isEqual),
-      shareReplay(1)
-    );
-  }
+  public readonly selectedJobs$: Observable<ExplorerJob[]> = this._selectedJobs$.pipe(
+    filter((v) => Array.isArray(v) && v.length > 0),
+    distinctUntilChanged(isEqual),
+    shareReplay(1)
+  );
 
-  private readonly _smvJobs$ = this.getSelectedJobs$().pipe(
+  public readonly singleMetricJobs$: Observable<ExplorerJob[]> = this._selectedJobs$.pipe(
     map((jobs) => jobs.filter((j) => j.isSingleMetricViewerJob)),
     shareReplay(1)
```

Comment:
Removed it completely since it was not used anywhere [#f725b4d](https://github.com/elastic/kibana/pull/203224/commits/f725b4daac45d744485f1015f2084310f03a2dc8)